.github/workflows/tests.yml

```
@@ -14,19 +20,27 @@ jobs:
+      run: composer install --ignore-platform-reqs --no-interaction --prefer-dist
+
 
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+      
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v2
 
-    - name: Build image
+    - name: Build image for ${{ matrix.platform }}
       uses: docker/build-push-action@v3
```

actionlint (1.7.9)

the runner of "docker/build-push-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)